flang/lib/Lower/PFTBuilder.cpp

```diff
@@ -770,6 +770,11 @@ class PFTBuilder {
             markSuccessorAsNewBlock(eval);
           },
 
+          // The first executable statement after an EntryStmt is a new block.
+          [&](const parser::EntryStmt &) {
+            eval.lexicalSuccessor->isNewBlock = true;
```

Comment:
Is it guaranteed that entry statements always have a lexical successor that can be a new block ?